quiz/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"bufio"
+	"encoding/csv"
+	"flag"
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"strings"
+	"time"
+)
+
+type Quiz struct {
+	filename       string
+	timeLimit      int
+	shuffle        bool
+	problems       [][]string
```

Comment:
It seems like a `Problem` or `Question` struct would be better than a slice of slices.